src/component/contents/DraftEditorLeaf.react.js

```diff
@@ -45,7 +45,7 @@ type Props = {
 
   // The current `SelectionState`, used to represent a selection range in the
   // editor
-  selection: SelectionState,
+  selection: ?SelectionState,
```

Comment:
Would it work to do `SelectionState | null`?

Reply:
We could do that, I have no strong opinion around it, we are currently doing a non strict check for it on https://github.com/facebook/draft-js/blob/master/src/component/contents/DraftEditorLeaf.react.js#L86